core/src/main/java/io/aiven/klaw/helpers/db/rdbms/SelectDataJdbc.java

```diff
@@ -1926,4 +1941,151 @@ public boolean existsConnectorRequest(
+      boolean isApproval,
+      String requestor,
+      String requestStatus,
+      boolean showRequestsOfAllTeams,
+      int tenantId,
+      Integer teamId,
+      OperationalRequestType operationalRequestType,
+      String env,
+      String wildcardSearch,
+      boolean isMyRequest) {
+    if (log.isDebugEnabled()) {
+      log.debug(
+          "selectFilteredOperationalRequests {} {} {} {} {} {}",
+          showRequestsOfAllTeams,
+          requestor,
+          requestStatus,
+          teamId,
+          env,
+          wildcardSearch);
+    }
+    Integer teamSelected = selectUserInfo(requestor).getTeamId();
+    List<OperationalRequest> operationalRequests = new ArrayList<>();
+    List<OperationalRequest> operationRequestListSub;
+    if (isApproval) { // approvers
+      // On Approvals this should always be filtered by the users current team
+      operationRequestListSub =
+          Lists.newArrayList(
+              findOperationalRequestsByExample(
+                  operationalRequestType,
+                  showRequestsOfAllTeams ? null : teamSelected,
+                  env,
+                  requestStatus,
+                  tenantId,
+                  null,
+                  isMyRequest ? requestor : null));
+
+      // remove users own requests to approve/show in the list
+      // Placed here as it should only apply for approvers.
+      operationRequestListSub =
+          operationRequestListSub.stream()
+              .filter(topicRequest -> !topicRequest.getRequestor().equals(requestor))
+              .toList();
+    } else {
+      if (showRequestsOfAllTeams) {
+        operationRequestListSub =
+            Lists.newArrayList(
+                findOperationalRequestsByExample(
+                    operationalRequestType,
+                    null,
+                    env,
+                    requestStatus,
+                    tenantId,
+                    null,
+                    isMyRequest ? requestor : null));
+      } else {
+        operationRequestListSub =
+            Lists.newArrayList(
+                findOperationalRequestsByExample(
+                    operationalRequestType,
+                    teamSelected,
+                    env,
+                    requestStatus,
+                    tenantId,
+                    null,
+                    isMyRequest ? requestor : null));
+      }
+    }
+
+    if (teamId != null) {
+      operationRequestListSub =
+          operationRequestListSub.stream()
+              .filter(topicRequest -> topicRequest.getRequestingTeamId().equals(teamId))
+              .toList();
+    }
+
+    boolean wildcardFilter = (wildcardSearch != null && !wildcardSearch.isEmpty());
+
+    for (OperationalRequest row : operationRequestListSub) {
+
+      try {
+        row.setRequesttimestring(
+            (new SimpleDateFormat("dd-MMM-yyyy HH:mm:ss"))
+                .format((row.getRequesttime()).getTime()));
```

Comment:
why can't we use `DateTimeFormatter` which is thread safe and could be declared in `static final` ?